aws-amplify/docs
# Update links for index pages

#### Description of changes:
- Update links for `/gen1/index.tsx` and `/gen1/[platform]/index.tsx` 
   - This includes the get started popover and the links for the features
- Updated the get started popover component to accept an array of objects used to display the links in the popover dropdown
- Updated the get started popover on the gen2 index pages to route to `/[platform]/start/quickstart/`

Staging site: https://update-links-gen1-index-pages.d1ywzrxfkb9wgg.amplifyapp.com/

To test:
1. Go to https://update-links-gen1-index-pages.d1ywzrxfkb9wgg.amplifyapp.com/gen1
2. Click around on the links in the main content and the get started popover to make sure they are linked correctly
3. https://update-links-gen1-index-pages.d1ywzrxfkb9wgg.amplifyapp.com/gen1/javascript
4. Repeat step 2
5. Go to https://update-links-gen1-index-pages.d1ywzrxfkb9wgg.amplifyapp.com/
6. Repeat step 2
7. Go to https://update-links-gen1-index-pages.d1ywzrxfkb9wgg.amplifyapp.com/swift/ or any other platform
8. Repeat step 2

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
